src/Servers/Kestrel/Core/src/Internal/Infrastructure/HttpUtilities.cs

```diff
@@ -411,7 +411,7 @@ public static string VersionToString(HttpVersion httpVersion)
         public static string MethodToString(HttpMethod method)
         {
             int methodIndex = (int)method;
-            if (methodIndex >= 0 && methodIndex <= 8)
+            if ((uint)methodIndex < (uint)_methodNames.Length)
```

Comment:
@AndyAyersMS should I take this `static readonly` array to a local? (thinking re: https://github.com/dotnet/coreclr/issues/21951)

Reply:
Probably so -- otherwise I would expect the to will redo the bounds check, since it will think `_methodNames` might change.

Reply:
👍